packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -351,7 +351,7 @@ contract Initializer_Test is Bridge_Initializer {
         // Ensure that all L1, L2 `Initializable` contracts are accounted for, in addition to
         // OptimismMintableERC20FactoryImpl, OptimismMintableERC20FactoryProxy, OptimismPortal2,
         // DisputeGameFactoryImpl, DisputeGameFactoryProxy, DelayedWETHImpl, DelayedWETHProxy.
-        assertEq(_getNumInitializable() + 5, contracts.length);
+        assertEq(_getNumInitializable() + 3, contracts.length);
```

Comment:
we have to make this change since `_getNumInitializable()` now picks up `OptimismPortalInterop` (counts +2 per contract, hence the subtraction from 5 to 3), because we don't extend `contracts` with `OptimismPortalInterop`. we cannot do that because `contracts` is constructed in the setup function of the test by looking up the contract in the deploy script (i.e., `deploy.mustGetAddress("OptimismPortal")`), for which does not exist a unique path for `OptimismPortalInterop` (it uses `deploy.mustGetAddress("OptimismPortal")`, so it would be a repetition of `OptimismPortal` if we wanted to add `OptimismPortalInterop` to `contracts` here)